src/raw/poll_evented.rs

```diff
@@ -377,13 +391,17 @@ where
     E: Evented,
     &'a E: Read,
```

Comment:
I think this might get covered by the blanket Pin implementation in futures  io since there is an implementation of AsyncRead for PollEvented<E: Read>